spec/toysh-posix.test.sh

```diff
@@ -10,7 +10,7 @@ abc=${a?bc} echo hello; echo blah
 ## stdout-json: ""
 
 #### setting readonly var (bash is only one where it's non-fatal)
-# http://landley.net/notes.html#20-06-2020
+# https://landley.net/notes-2020.html#12-06-2020
```

Comment:
Hm  adding `-2020.html` is right, but the anchor still starts with `#20-06`

https://landley.net/notes-2020.html#20-06-2020